source/_docs/installation/virtualenv.markdown

```diff
@@ -23,33 +30,54 @@ $ sudo apt-get install python3-pip python3-dev
 $ sudo pip3 install --upgrade virtualenv
 ```
 
-### {% linkable_title Step 1: Create a Home Assistant user & group %}
+## {% linkable_title Step 2: Setup virtualenv %}
+
+```bash
+$ python3 -m venv $HOME/homeassistant
+```
+
+## {% linkable_title Step 3: Install or update Home Assistant %}
+
+```bash
+$ source $HOME/homeassistant/bin/activate
+(homeassistant)$ pip3 install --upgrade homeassistant
+```
+
+## {% linkable_title Step 4: Run Home Assistant %}
+
+```bash
+$ $HOME/homeassistant/bin/hass
+```
+
+### {% linkable_title Advanced guide %}
+## {% linkable_title Separate user & group for Home Assistant (Basic guide step 2) %}
 
-This step is optional, but it's a good idea to give services like Home Assistant their own user. It gives you more granular control over permissions, and reduces the exposure to the rest of your system in the event there is a security related bug in Home Assistant. This is a reasonably Linux oriented step, and will look different on other operating systems (or even other Linux distributions).
+It's a good idea to give services like Home Assistant their own user. It gives you more granular control over permissions, and reduces the exposure to the rest of your system in the event there is a security related bug in Home Assistant. This is a reasonably Linux oriented step, and will look different on other operating systems (or even other Linux distributions).
 
 ```bash
 $ sudo adduser --system homeassistant
 $ sudo addgroup homeassistant
 ```
 
-Home Assistant stores its configuration in `$HOME/.homeassistant` by default, so in this case, it would be in `/home/homeassistant/.homeassistant`
+Home Assistant stores it's configuration in `$HOME/.homeassistant` by default, so in this case, it would be in `/home/homeassistant/.homeassistant`.
```

Comment:
It should be `its`. Here it should be a possessive pronoun.

Reply:
😢 thanks... I should refresh my english